Packs/CuckooSandbox/Integrations/integration-CuckooSandbox.yml

```diff
@@ -148,51 +148,79 @@ script:
         url = fixUrl(qParams['server']) + suffix;
         username = qParams['authentication'] ? qParams['authentication']['identifier'] : '';
         password = qParams['authentication'] ? qParams['authentication']['password'] : '';
+        full_api_key = '';
+        if (username == '__token'){
+            full_api_key = 'Bearer ' + password;
+            username = '';
+            passwoed = '';
+        }
         res = http(
-            url,
-            {
-                Method: 'GET',
-                Username: username,
-                Password: password
-            },
-            qParams['insecure'],
-            qParams['proxy']
-        );
+              url,
+              {
+                  Method: 'GET',
+                  Headers: {
+                      'Authorization': [full_api_key]
+                  },
+                  Username: username,
+                  Password: password
+              },
+              qParams['insecure'],
+              qParams['proxy']
+          );
```

Comment:
is it possible to move this logic to a separate function `req()` and call it from every function that makes an API call?